src/ngraph/runtime/gpu/unit_test.manifest

```diff
@@ -53,6 +53,7 @@ quantize_int8
 quantize_clamp
 dequantize
 dequantize_axes
+dequantize_cpu
```

Comment:
This test is not a generalized backend unit test (not NGRAPH_TEST) so it does not need to be in the manifest